manifests/plugin/puppetdb.pp

```diff
@@ -13,10 +13,25 @@
 # $dashboard_address:: Address of puppetdb dashboard. Defaults to 'http://localhost:8080/dashboard'
```

Comment:
This (and ```$address``` above) is no longer correct. Maybe we should just remove the Defaults to part since the installer will show the default value anyway.